src/NLog/Layouts/JsonLayout.cs

```diff
@@ -108,12 +108,12 @@ protected override void InitializeLayout()
             base.InitializeLayout();
             if (IncludeMdc)
             {
-                ThreadAgnostic = false;
+                base.ThreadAgnostic = false;
             }
-#if !SILVERLIGHT && !NETSTANDARD || NETSTANDARD1_3
```

Comment:
If you remove NETSTANDARD then IncludeMdlc is supported by all platforms except SILVERLIGHT. So I guess it should be:

#if !SILVERLIGHT